vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -549,13 +549,6 @@ void AdsImpl::Start() {
 
   LogActiveStudies();
 
-#if BUILDFLAG(IS_ANDROID)
```

Comment:
why do we remove this code?

Reply:
This code was moved to notification ad manager, thanks